tests/suites/test_suite_pk.function

```diff
@@ -1118,16 +1104,15 @@ void pk_rsa_verify_ext_test_vec(data_t *message_str, int digest,
          * PSA or the Mbed TLS API, depending on the PSS options used.
          * So, it may return either INVALID_PADDING or INVALID_SIGNATURE.
          */
+        fprintf(stderr, "result=%d, ret=%d\n", result, ret);
```

Comment:
Leftover from debugging.

Also, looks like we could just update the data file with the correct error code now that we're always using PSA, and then we no longer need this `if` and comment here.